### ISSUE TITLE
watch tries to load stylesheet even though I haven't asked it to; breaks.

### ISSUE THREAD

**Issue by garethrees** [jdan/cleaver#136]
https://github.com/jdan/cleaver/blob/master/bin/cleaver#L78-L80

```sh
$ cleaver watch blah.md
Watching for changes on blah.md. Ctrl-C to abort.
Also watching for changes on undefined
path.js:439
      throw new TypeError('Arguments to path.resolve must be strings');
            ^
TypeError: Arguments to path.resolve must be strings
    at Object.posix.resolve (path.js:439:13)
    at Object.fs.watchFile (fs.js:1256:25)
    at Command.<anonymous> (/usr/local/lib/node_modules/cleaver/bin/cleaver:81:10)
    at Command.<anonymous> (/usr/local/lib/node_modules/cleaver/node_modules/commander/index.js:249:8)
    at Command.emit (events.js:110:17)
    at Command.parseArgs (/usr/local/lib/node_modules/cleaver/node_modules/commander/index.js:477:12)
    at Command.parse (/usr/local/lib/node_modules/cleaver/node_modules/commander/index.js:370:21)
    at Object.<anonymous> (/usr/local/lib/node_modules/cleaver/bin/cleaver:133:9)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
```

**Comment by jdan** [comment]
Thanks for reporting @garethrees. I went ahead and published a version bump (0.7.6), so once you update things should stop breaking as fiercely. Sorry!

**Comment by garethrees** [comment]
:+1: thanks!
